paddle/optimizer/adagrad_optimizer.cc

```diff
@@ -0,0 +1,4 @@
+#include "sgd_optimizer.h"
+
+namespace paddle {
```

Comment:
This is not a working source file, probably checked in by mistake. Maybe it's a good idea to review the PR first so that these mistakes can be avoided.